### DISCUSSION TITLE
How to Use SQL Functions (MAX, MIN, AVG) in Query

### DISCUSSION THREAD

**Discussion by rida-re** [medusajs/medusa#9777] [Q&A]
Hi everyone!

I'm currently working on a Medusa project v2 and need some help with using SQL aggregate functions like MAX, MIN, and AVG to query the database and select specific columns. I've been exploring ways to achieve this, but I'm unsure about how to implement these functions in my query using https://docs.medusajs.com/learn/advanced-development/module-links/query

Any advice or examples on how to correctly use these SQL operators, would be greatly appreciated!

Thanks in advance for your help :)

**Comment by gajeraharsh** [comment]
Did you find any solution for this ?

**Reply by rida-re** [reply]
I did not find a solution

**Comment by docloulou** [comment]
https://docs.medusajs.com/learn/fundamentals/modules/db-operations#run-queries

You can use raw sql with this
